bigchaindb/upsert_validator/validator_election.py

```diff
@@ -220,6 +220,7 @@ def get_validator_update(cls, bigchain, new_height, txns):
                 updated_validator_set = new_validator_set(curr_validator_set,
                                                           new_height, validator_updates)
 
+                updated_validator_set = [v for v in updated_validator_set if v['voting_power'] > 0]
```

Comment:
When do you expect the voting power to be zero here?

Reply:
When someone wishes to remove a validator they will set the validator power to `0`

Reply:
Should we not allow them doing that?

Reply:
Do you wish to imply that we should not allow users to remove the validators?

Reply:
I believe, this restriction does not make much sense. Imagine a network where for some reason one thousand is a common voting power. Setting a voting power of 1 is then effectively the same as 0 so we as well just allow 0 to be voted for.